### DISCUSSION TITLE
Is it a good idea to run a watchtower backed by neutrino?

### DISCUSSION THREAD

**Discussion by hazrulnizam** [lightningnetwork/lnd#6196] [Q&A] [answered]
Having a watchtower in a different geographical location to your lnd node have obvious benefits, and this can be done in a cost-effective manner using VPS services. However, most VPS service do not include enough storage space to enable running a full bitcoin node on them. Using Neutrino-backed lnd as watchtower seems like a solution, as you don't need the huge storage requirements to store the entire blockchain.

Is there any fatal drawbacks to this approach?

**Comment by bota87** [comment]
I'm asking the same and if neutrino isn't good for that what about a pruned bitcoind?

**Comment by guggero** [comment, marked as answer]
Since a watchtower mostly looks at new, incoming blocks, I'd actually suggest using a pruned `bitcoind` node. The slight disadvantage of a Neutrino node vs. a pruned `bitcoind` node is that Neutrino doesn't look at the mempool. But because AFAIK a watchtower only reacts after a breach TX was confirmed, this shouldn't make a big difference.
So Neutrino should work well too. Just make sure you specify multiple peers or your own node that reliably serves you blocks. If you end up not getting new blocks because your Neutrino client isn't served anymore, your watchtower can't watch the chain.

**Reply by hazrulnizam** [reply]
Got it! That's very helpful, thanks!

**Comment by fbrz76** [comment]
Hi, 
i am taking advantage of this discussion for a question about neutrino:
i have a neutrino-enabled bitcoin full node configured as shown in this document: https://github.com/lightninglabs/docs.lightning.engineering/blob/master/lightning-network-tools/lnd/enable-neutrino-mode-in-bitcoin-core.md 
my question is? what port must be made available on the full node server for the neutrino-based lnd client to connect ? I can't find any indication in the logs to help me understand.

i think the "p2p port" is the 8333 by default. right ?

Thanks

**Reply by guggero** [reply]
Yes, the p2p port is the one you want. The Neutrino client uses normal p2p messages to fetch the filters from your full node. Be aware that opening that port to the public might cause other Neutrino clients to download filters from your node, causing quite a bit of traffic. You can limit the number of inbound connection to somewhat limit that amount of traffic indirectly.
